docs/src/content/docs/guides/make-onetime-payment.mdx

```diff
@@ -121,28 +98,140 @@ Add the following properties:
   chunk='3'
 />
 
-### 7. Request an Outgoing Payment grant
+### 5. Request an interactive Outgoing Payment grant
 
-Request an `outgoingPayment` grant from the sender wallet's authorization server.
+Request an `outgoingPayment` grant from the sender wallet's authorization server. This request requires an interactive grant as the sender will need to consent before an `outgoingPayment` is made against their wallet.
 
 Add the `limits` object with the following properties:
 
 - `debitAmount` : the maximum amount to be deducted from the sender's wallet.
 - `receiveAmount` : the maximum amount to be received in the recipient's wallet.
 
-:::note
-This request requires an interactive grant as the sender will need to consent before an `outgoingPayment` is made against their wallet.
+:::note[Limits]
+
+The optional `debitAmount` and `receiveAmount` parameters, which apply to the sender and recipient respectively, allow you to define the limits under which an outgoing payment can be created.
 
-For details on how to facilitate an interactive grant refer to the <LinkOut href='https://openpayments.dev/guides/create-interactive-grant/'>
-Create an Interactive Grant guide. </LinkOut>
+The value for either parameter is the amount that all payments created under the grant cannot exceed. For example, the `DEBIT_AMOUNT` value is the maximum amount that can be debited from the payer's wallet.
```

Comment:
We missed one :) From the sender's wallet, not the payer's wallet.

Also, I'm not sure that we should mention "The value...is the amount that all payments created under the grant..." because there should only be one payment under this grant. Might want to check with Max or Blair.